modules/luci-mod-status/htdocs/luci-static/resources/view/status/syslog.js

```diff
@@ -21,16 +21,37 @@ return view.extend({
 	render: function(logdata) {
 		var loglines = logdata.trim().split(/\n/);
 
+
+		var scrollDownButton = E('button', { 
+				'id': 'scrollDownButton',
+				'class': 'cbi-button cbi-button-neutral'
+			}, _('Scroll down')
+		);
+		scrollDownButton.addEventListener('click', function() {
+			window.scrollTo(0, document.body.scrollHeight);
+		});
+
+		var scrollUpButton = E('button', { 
+				'id' : 'scrollUpButton',
+				'class': 'cbi-button cbi-button-neutral'
+			}, _('Scroll up')
```

Comment:
You could change instances to just `Tail` - a simpler way of saying more concretely `Scroll to bottom` or `Scroll to tail`, which is more definitive than just `Scroll down`.

Likewise for the other button names. You decide, but put something more concrete.

Reply:
To understand correctly, you just want to name the button "tail" and "head"? If you think that is more direct then im fine with that.

Reply:
It's short and easier to translate than three words. :)

Maybe `Scroll to tail` is best - more context for the i18n people.

Reply:
yeah i can do that, i can also add a translation hint separately.